src/js/api/config.js

```diff
@@ -39,6 +39,8 @@ export const Defaults = {
     playbackRates: [0.5, 1, 1.25, 1.5, 2],
     renderCaptionsNatively: false,
     repeat: false,
+    // loop is only used on native html5 providers to loop mp4
+    loop: false,
```

Comment:
config is specific to anything a user can configure. You can store this attribute on the model without extending config.